app/controllers/application_controller.rb

```diff
@@ -3,12 +3,6 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
-  helper_method :current_division
-
-  def current_division
```

Comment:
`current_division`, or at least a version of it, needs to be in the `ApplicationController`, the public side driving the wordpress stuff is dependent on it.